utils/onboarding/weblog_interface.py

```diff
@@ -5,9 +5,15 @@
 
 
 def make_get_request(app_url):
-    generated_uuid = str(randint(0, 100000000000000000))
+    generated_uuid = str(randint(1, 100000000000000000))
```

Comment:
0 gets dropped by the backend, even if on paper the probability is 0, avoiding this case to happen altogether